languages/scala/src/test/scala/SampleClassTest.java

```diff
@@ -0,0 +1,63 @@
+import org.junit.jupiter.api.Test;
```

Comment:
I chose to implement tests in vanilla Java/JUnit to see how it'd interop with our target Scala classes. Everything worked smoothly!